engines/wage/wage.cpp

```diff
@@ -605,5 +627,9 @@ void WageEngine::processTurn(Common::String *textInput, Designed *clickInput) {
 	_inputText.clear();
 }
 
+Common::Language WageEngine::getLanguage() const{
```

Comment:
```suggestion
Common::Language WageEngine::getLanguage() const {
```

Please follow our code formatting guidelines. Adn you may see this formatting _everywhere_